Bash_scripts/HW28/shellcheck.sh

```diff
@@ -6,3 +6,4 @@ SHELL_SCRIPTS=$(find "${1:-.}" -name '*.sh')
 for var in $SHELL_SCRIPTS ; do
 shellcheck "$var"
 done
+
```

Comment:
?

Reply:
this is not necessary) at first I wanted to use the file with the script, but then changed my mind)   Deleted